content/en/docs/deployment/installation/jwe.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "JWE Key generator tool installation"
+linkTitle: "JWE_Generator"
+weight: 4
+---
+Install JWE key generator tool for the Axway Open Banking solution.
+
+## Download the JWE Helm chart
+
+Download the Axway Open Banking JWE Tool Helm chart to customize it locally.
+
+```bash
+helm pull open-banking/jwe-generator --untar
+```
+
+You should get an `jwe-generator` local folder.
+
+## Customize the JWE key generator tool Helm chart
+
+Customize the `jwe-generator/values.yaml` file as follows.
+
+| Value         | Description                           | Default value  |
+|:------------- |:------------------------------------- |:-------------- |
+| ingress.domainName | Set the domainname for all ingress. | \<domain-name> |
+| global.dockerRegistry.username | Login name to pull Docker images from the Axway Repository. | None |
+| global.dockerRegistry.token | Password token to pull Docker images from the Axway Repository. | None |
+
+## Install the JWE key generator Helm chart
+
+1. Create the target namespace on the cluster:
+
+   ```bash
+   kubectl create namespace jwe-generator
+   ```
+
+2. Install the Analytics Helm charts:
```

Comment:
```suggestion
2. Install the JWE key generator Helm charts:
```
Also, I don't know if it is one chart or more... if this should be "charts" or "chart."